kobo/apps/stripe/tests/test_link_creation_api.py

```diff
@@ -55,7 +55,6 @@ def test_rejects_invalid_query_params(
         url = self._get_url({'price_id': 'test', 'organization_uid': 'test'})
         response = self.client.post(url)
         assert response.status_code == status.HTTP_400_BAD_REQUEST
-        assert response.data['price_id'].count('Invalid price ID') == 1
```

Comment:
It would be good to add a comment in the Pull Request as to why this was done. But no code comment is needed.

Reply:
@LMNTL I am guessing this was one of your changes?

Reply:
Yes! I refactored `CheckoutLinkView` and its serializer, which changed the error that would get produced and broke that test. Then I realized the test case was too narrow (really we just care that we got the right status) so I removed that assertion.